pkg/hive/apiserver/registry/util/convert.go

```diff
@@ -0,0 +1,591 @@
+package util
+
+import (
+	netopv1 "github.com/openshift/cluster-network-operator/pkg/apis/networkoperator/v1"
+	hivev1 "github.com/openshift/hive/pkg/apis/hive/v1"
+	"github.com/openshift/installer/pkg/ipnet"
+	installtypes "github.com/openshift/installer/pkg/types"
+	installtypesaws "github.com/openshift/installer/pkg/types/aws"
+	installtypesazure "github.com/openshift/installer/pkg/types/azure"
+	installtypesgcp "github.com/openshift/installer/pkg/types/gcp"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	hiveapi "github.com/openshift/hive/pkg/hive/apis/hive"
```

Comment:
I'm confused about these types vs pkg/apis/hive/v1alpha1, the comments here indicate we're returning v1alpha1 types in this package.